src/Features/Core/Portable/InitializeParameter/AbstractInitializeParameterCodeRefactoringProvider.cs

```diff
@@ -29,18 +29,27 @@ internal abstract partial class AbstractInitializeParameterCodeRefactoringProvid
         where TStatementSyntax : SyntaxNode
         where TExpressionSyntax : SyntaxNode
     {
-        private readonly Func<SyntaxNode, bool> _isFunctionDeclarationFunc;
+        protected readonly Func<SyntaxNode, bool> _isFunctionDeclarationFunc;
+        protected readonly Func<SyntaxNode, bool> _isRecordDeclarationFunc;
```

Comment:
With the compiler change about caching method groups, do we need to bother with the protected fields any more, versus just referencing the methods directly?

Reply:
This is not a static method, so we'd still need this manual caching